larsim/LegacyLArG4/OpFastScintillation.cxx

```diff
@@ -1577,7 +1577,7 @@ namespace larg4 {
 
     // detemine hits on each PD
     const std::array<double, 3> hotspot = {plane_depth, ScintPoint.Y(), ScintPoint.Z()};
-    for (size_t const OpDet : util::counter(NOpChannels)) {
+    for (size_t const OpDet : util::counter(fPVS->NOpChannels())) {
```

Comment:
Yes, thanks, done with a4c7c2e.

Reply:
It removed the `NOpChannels` member because it was not initialized in the constructor, and so depending on how you call this class you may not get `NOpChannels` properly initialized. I can either move the initialization to the constructor, or just remove this variable as the number of op dets is provided by the service. I preferred to go the second way as I thought it was cleaner.

Reply:
OK, sounds good good.